porting/first-steps.rst

```diff
@@ -68,7 +68,7 @@ Install the required dependencies::
      libx11-dev:i386 libreadline6-dev:i386 libgl1-mesa-glx:i386 \
      libgl1-mesa-dev g++-multilib mingw-w64-i686-dev tofrodos \
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
-     repo liblz4-tool bc lzop imagemagick
+     repo liblz4-tool bc lzop imagemagick libncurses5 rsync
```

Comment:
You also add rsync to the requirements. Are you sure it's neccesary? If you are, please explain it in the pull request description as well.

Reply:
Hi JBBgameich,
I was building in a Debian chroot, and mka systemimage was failing due to rsync missing.